pkgs/development/python-modules/beaker/default.nix

```
@@ -19,14 +19,14 @@
 
 buildPythonPackage rec {
   pname = "Beaker";
-  version = "1.10.1";
+  version = "1.11.0";
 
   # The pypy release do not contains the tests
   src = fetchFromGitHub {
     owner = "bbangert";
     repo = "beaker";
     rev = version;
-    sha256 = "0xrvg503xmi28w0hllr4s7fkap0p09fgw2wax3p1s2r6b3xjvbz7";
+    sha256 = "sha256-EragdCvugAGW0/UN8HP0nbnWio7t3pq8YBSkrOJhOhU=";
```

```suggestion
    sha256 = "059sc7iar90lc2y9mppdis5ddfcxyirz03gmsfb0307f5dsa1dhj";
```
I would prefer not to have sri hashes, for supporting older versions of nix cli

```
$ nix to-base32 sha256-EragdCvugAGW0/UN8HP0nbnWio7t3pq8YBSkrOJhOhU=
059sc7iar90lc2y9mppdis5ddfcxyirz03gmsfb0307f5dsa1dhj
```